.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to vaccineff
```

Comment:
You can also link to the organisation wide contributing document instead of having one in the package which may become outdated

Reply:
Same here. Addressed in #88